app/policies/template_policy.rb

```diff
@@ -1,7 +1,10 @@
 class TemplatePolicy < ApplicationPolicy
   class Scope < ApplicationPolicy::Scope
     def resolve
```

Comment:
This implementation is the same as for workflow policy. We can make it a default implementation in `ApplicationPolicy::Scope`.